### DISCUSSION TITLE
Expose more verbose details for debugging?

### DISCUSSION THREAD

**Discussion by t829702** [flareact/flareact#120] [General]
When I already have something else running on localhost:8080 ;  this `Starting Flareact dev server on http://localhost:8080` message is a silent failure:  accessing via http://localhost:8080 reached my other server, but via http://127.0.0.1:8787 is ok,

```console
➸ npm run dev
🚀 Starting Flareact dev server on http://localhost:8080 ...

[worker] 🌀  Using namespace for Workers Site "__my-flareact-project-workers_sites_assets_preview"
[worker] ✨  Success
[worker] 👂  Listening on http://127.0.0.1:8787
[worker] 🌀  Detected changes...
[worker] 💁  Ignoring stale first change
[worker] [2021-02-10 09:08:50] GET my-flareact-project.peregrine.workers.dev/dashboard HTTP/1.1 500 Internal Server Error
```

Til I killed the other thing running localhost:8080, I can see the service running on localhost:8080, it's just same as on 127.0.0.1:8787

```console
➸ time npm run dev

> flareact-template@1.0.0 dev
> flareact dev

🚀 Starting Flareact dev server on http://localhost:8080 ...

[worker] 🌀  Using namespace for Workers Site "__my-flareact-project-workers_sites_assets_preview"
[worker] ✨  Success
[worker] 👂  Listening on http://127.0.0.1:8787
[worker] 🌀  Detected changes...
[worker] [2021-02-10 09:24:45] GET my-flareact-project.peregrine.workers.dev/dashboard HTTP/1.1 500 Internal Server Error
[worker] [2021-02-10 09:24:45] GET my-flareact-project.peregrine.workers.dev/cdn-cgi/styles/cf.errors.css HTTP/1.1 200 OK
```

When I started development a page under `./page/dashboard.js` and when it's good, it's showing everything normal; but it contains some errors, the page is only showing Error 1101 with Ray ID, and `An unknown error occurred` does not help debugging, and it wants me login to Cloudflare and check the error logs, but at the moment, this service is not published yet, there's nothing in Cloudflare console,

and from dev terminal: there's no error logs beyond the 500 Internal Server Error line:

```
[worker] [2021-02-10 09:24:45] GET my-flareact-project.peregrine.workers.dev/dashboard HTTP/1.1 500 Internal Server Error
```

```
Error 1101 Ray ID: 61f7868b19d5025e • 2021-02-10 17:24:45 UTC
Worker threw exception
What happened?

What happened?
You've requested a page on a website (my-flareact-xxxxxx.xxxxx.workers.dev) that is on the Cloudflare network. An unknown error occurred while rendering the page.

What can I do?
If you are the owner of this website:
you should login to Cloudflare and check the error logs for my-flareact-xxxxxx.xxxxx.workers.dev......
```

![image](https://user-images.githubusercontent.com/49542456/107547741-a69a9080-6b82-11eb-88d2-c1a5789868bc.png)

**Comment by traffisco** [comment]
Workers are not too verbose with exceptions... 
You can do the classic try catch blocks with logging. The struggle is real.
